shared/test/views_test.js

```diff
@@ -604,7 +604,7 @@ describe("loop.shared.views", function() {
       });
 
       it("should not throw if the element is not a video object", function() {
-        sinon.stub(view, "getDOMNode").returns({
+        sinon.stub(ReactDOM.findDOMNode(view)).returns({
```

Comment:
Oh, and these need to become sandbox.stub since we're stubbing a "global" rather than a created item.